tests/parsers/fixtures/pw/magnetic_moments_v68/aiida.out

```diff
@@ -0,0 +1,835 @@
+
+     Program PWSCF v.6.8 starts on 10Sep2021 at 18:59:30 
+
+     This program is part of the open-source Quantum ESPRESSO suite
```

Comment:
Think we can snip the kpoint information and the timing at the end of this file to reduce the new lines added somewhat.